test/src/test/java/org/hiero/mirror/test/e2e/acceptance/client/AbstractNetworkClient.java

```diff
@@ -204,6 +205,9 @@ public void validateAddress(final String actualAddress) {
                 .isEqualTo(
                         account.getPublicKey().isECDSA()
                                 ? account.getPublicKey().toEvmAddress().toString()
-                                : account.getAccountId().toSolidityAddress());
+                                : asAddress(account.getAccountId())
+                                        .toString()
+                                        .toLowerCase()
```

Comment:
If we create separate helper methods for string addresses and use that everywhere then we don't have to call `toString().toLowerCase()` everywhere and don't have to change them again later.

```suggestion
                                : asHexAddress(account.getAccountId())
```

Reply:
Added asHexAddress(AccountId accountId)

Reply:
I guess I misunderstood your comment. Apologies. It is not needed to change the asAddress(...).toString(). I've tested with the proposed fix in the TestUtil comment - to change the asAddress methods to utilize toSolidityAddress and the tests pass without the need to revert anything.